text/machine_comprehension/roberta/dependencies/benchmark_result.csv

```diff
@@ -0,0 +1,5 @@
+model_name,inputs,engine,version,device,precision,optimizer,io_binding,b1_s8,b1_s16,b1_s32,b1_s64,b1_s128,b4_s8,b4_s16,b4_s32,b4_s64,b4_s128

```

Comment:
Let's include the important details from this table inline in the metrics section of the readme

Reply:
Maybe we need not include benchmark result for the model. If we decided to include performance result, it shall also have hardware/software (like CPU, OS etc) description since the result will be different in different environment.